components/resources/wallet_strings.grdp

```diff
@@ -126,19 +131,23 @@
   <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_NO" desc="Create Account Tab No Button">No</message>
   <message name="IDS_BRAVE_WALLET_BUTTON_CONTINUE" desc="General Continue button">Continue</message>
   <message name="IDS_BRAVE_WALLET_BUTTON_NEXT" desc="General Next button">Next</message>
-  <message name="IDS_BRAVE_WALLET_BUTTON_GOT_IT" desc="Acknowledgement Next button">Got it</message>
+  <message name="IDS_BRAVE_WALLET_BUTTON_GOT_IT" desc="Acknowledgement Next button">View my recovery phrase</message>
```

Comment:
This sounds a bit odd to me. Maybe `Show my recovery phrase` or `View your recovery phrase`? cc: @rmcfadden3

Reply:
Sure, `Show my recovery phrase` seems fine here.